src/cairoUtilFuncGen/storage/mappingIndexAccess.ts

```diff
@@ -212,23 +213,23 @@ export class MappingIndexAccessGen extends CairoUtilFuncGenBase {
     const helperFuncName = `WS_TO_FELT_ARRAY${this.stringHashFunctions.size}`;
     return {
       name: funcName,
-      code: [
-        `fn ${helperFuncName}(name: felt, ref ptr: Array::<felt>, index: u256, len: u256){`,
-        `    if index == len{`,
-        `        return;`,
-        `    }`,
-        `    let loc = ${arrayName}::read(name, index);`,
-        `    let value = WARP_STORAGE::read(loc);`,
-        `    ptr.append(value);`,
-        `    ${helperFuncName}(name, ref ptr, index + 1, len)`,
-        `}`,
-        `fn ${funcName}(name: felt) -> felt {`,
-        `    let len: u256 = ${lenName}::read(name);`,
-        `    let mut ptr = ArrayTrait::new();`,
-        `    ${helperFuncName}(name, ref ptr, 0, len);`,
-        `    string_hash(len, ptr)`,
-        `}`,
-      ].join('\n'),
+      code: endent`
+          fn ${helperFuncName}(name: felt, ref ptr: Array::<felt>, index: u256, len: u256){
+            if index == len{
```

Comment:
Usually a space is put before `{`

Reply:
Usually where :eyes:. Have we achieved a consensus con Cairo coding style. I prefer it without (